### DISCUSSION TITLE
how i can add new language rtl

### DISCUSSION THREAD

**Discussion by vahidalvandi** [bagisto/bagisto#5974] [Q&A] [answered]
i want to add pesian in admin panel

**Comment by Sanjaybhattwebkul** [comment]
Hi @vahidalvandi 
Are you asking about the **Farsi** locale ??

**Comment by vahidalvandi** [comment]
@Sanjaybhattwebkul yes

**Comment by Sanjaybhattwebkul** [comment, marked as answer]
@vahidalvandi 
Please take a reference from [Here](https://prnt.sc/fmPtippkgxxL). For adding the Farsi locale

**Comment by vahidalvandi** [comment]
have bug in sidebar menu
![image](https://user-images.githubusercontent.com/7871623/157614607-bc81eabe-f14b-4589-b660-c457ce9ced09.png)

**Reply by vahidalvandi** [reply]
in RTL mode

**Comment by Sanjaybhattwebkul** [comment]
@vahidalvandi 
**Which version** of bagisto are you using?

**Reply by vahidalvandi** [reply]
1.3.3

**Comment by Sanjaybhattwebkul** [comment]
@vahidalvandi 
This **issue is fixed. in master** and will be **reflected in the next release.**
**For now,** you can take a **reference** from [Here](https://github.com/bagisto/bagisto/blob/master/packages/Webkul/Admin/src/Resources/assets/sass/partials/_rtl.scss#L53)

Complete code : https://github.com/bagisto/bagisto/pull/5343

**Reply by vahidalvandi** [reply]
I am very happy to meet bagisto and I hope we can develop this project more day by day.

**Reply by saurav-webkul** [reply]
It's great to work with you @vahidalvandi and yes we are making progress each day. 

Would love to meet you too someday.

**Reply by vahidalvandi** [reply]
I have been writing private cms with Laravel for 5 years and I am still working in Iran. I checked the bagisto architecture and I liked it and I hope I can use bagisto in new projects
